kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala

```diff
@@ -78,7 +78,12 @@ class FlinkProcessBuilder(
 
   override protected def commands: Array[String] = Array(executable)
 
-  override def killApplication(line: String = lastRowsOfLog.toArray.mkString("\n")): String = {
+  override def killApplication(clue: Either[String, String]): String = clue match {
```

Comment:
Why do we need this change ? For Spark SQL engine, we can call `killApplication(engineRefId)` and others call `killApplication()`

Reply:
And override the `killApplication` in `SparkProcessBuilder`.  BTW, we can rename the input parameter of `killApplication` to engineRefId

Reply:
> ... and others call `killApplication()`

Then it is not testable

Reply:
> ... and others call `killApplication()`

Then it is not testable